### ISSUE TITLE
Exam Calendar zeigt nichts an

### ISSUE THREAD

**Issue by sevorl** [JonasJoKuJonas/homeassistant-WebUntis#160]
### The problem

Ich habe den Exam-Calender natürlich direkt ausprobiert, Super Feature, das unseren Familienalltag sicher erleichtert.

Leider bekomme ich Prüfungen nicth angezeigt, die ich n der Untis-App sehe.

Ich weiß leider nicht genau, was ich zum reproduzieren sinnvoll beitragen kann.
Welche Informationen wären hilfreich?

### Version of Home Assistant?

2024.10.1

### What type of installation are you running?

Home Assistant OS

### Version of WebUntis

v.1.3.0

**Comment by JonasJoKuJonas** [comment]
@sevorl zeigt er bei dir auf der WebUntis Webseite auch nicht den Exam tab an, wie bei @cofw2005 ([(ink](https://github.com/JonasJoKuJonas/homeassistant-WebUntis/issues/100#issuecomment-2407187493))

**Comment by sevorl** [comment]
Genau, ich sehe keinen Exam-Tab. Meine Ansicht sieht so (ähnlich) aus wie von cofw2005. 
Kann es sein, dass es die Prüfungs-Ansicht nur für Lehrer gibt?

**Comment by svewag** [comment]
Ich hab mir von der Website mal den homework request als cURL kopiert und den Endpunkt dann mit `exams` ersetzt. Die Header inkl. `cookie` hab ich beibehalten.
Als Ergebnis bekomme ich ein 403.

`curl 'https://nessa.webuntis.com/WebUntis/api/exams?startDate=20241001&endDate=20241031'`

`{"errors":[{"code":"INTERNAL_ERROR","title":"Unerlaubter Zugriff"}]}%`

Ich sehe den Tab auch nicht.

<img width="274" alt="image" src="https://github.com/user-attachments/assets/7d51c098-4dbc-43d6-8b3f-10f71b68086e">

**Comment by JonasJoKuJonas** [comment]
> Kann es sein, dass es die Prüfungs-Ansicht nur für Lehrer gibt?

Nein. Bei mir wird die ja angezeigt.
![image](https://github.com/user-attachments/assets/df5eb20e-1f34-4dec-b2da-f1901a633007)
ich vermute, das es mit der Konfiguration von der Schule zusammenhängt.

@svewag hast du mehrere Accounts, weil bei dir Erziehungsberechtigte*r steht?

**Comment by JonasJoKuJonas** [comment]
Ihr könntet mal probieren diesen code mit euren credentials auszuführen. Ich bekomme da nur ein 
`webuntis.errors.RemoteError: no right for getExams()` Error

```
import webuntis
from datetime import date, timedelta


s = webuntis.Session(
    username="",
    password="",
    server="https://",
    useragent="foo",
    school="",
).login()


today = date.today()
in_x_days = today + timedelta(days=30)


x = s.exams(start=today, end=in_x_days)

for i in x:
    print(i)

```

**Comment by cofw2005** [comment]
Looks like the schools configure the Untis differently. So unfortunately not all schools have this Exam function.
Maybe this is also the same for the home function for some other schools.

**Comment by cofw2005** [comment]
I see so far at least 'Dienst' and 'IServ' tabs from others which I don't have in my school. So some functions are really school-dependent.

**Comment by JonasJoKuJonas** [comment]
You can also try to find out the URL that is used to fetch the exams by using the chrome debug menu (Network Tab)

**Comment by cofw2005** [comment]
just a suggestion, maybe a configuration option can be added to let user select whether the exam calendar and homework calendar shall be activated. So for the ones whose school don't have that option can disable the calendar entity.

**Comment by jkadw** [comment]
Hi Jonas, ich habe mal im Browser etwas genauer reingeschaut. Auch bei meinem Sohn gibt es keinen Extra-Reiter "Prüfungen". Allerdings bekommt man eine Zusatzinfo beim Abfragen des Kalenders:
https://nessa.webuntis.com/WebUntis/api/rest/view/v1/timetable/entries?start=2025-09-22&end=2025-09-26&format=1&resourceType=STUDENT&resources=6524&periodTypes=&timetableType=MY_TIMETABLE

                {
                    "ids": [
                        3526797
                    ],
                    "duration": {
                        "start": "2025-09-22T11:40",
                        "end": "2025-09-22T12:25"
                    },
                    "type": "EXAM",
                    "status": "REGULAR",
                    "statusDetail": null,
                    "name": null,
                    "lessonText": "",
                    "lessonInfo": "Klassenarbeit Deutscharbeit",
                    ...
              }
Bei normalen Stunden steht "type": "NORMAL_TEACHING_PERIOD". Die Info über die Art der Arbeit steht in "lessonInfo".

Man könnte in diesem Fall also den Exam-Kalender per Filter auf diesen Typ befüllen.
